src/components/pages/CaseTable/CaseTable.js

```diff
@@ -2,38 +2,35 @@ import React, { useState } from 'react';
 import axios from 'axios';
 import { Link, useParams } from 'react-router-dom';
 import Plot from 'react-plotly.js';
-import Tabs from '@material-ui/core/Tabs';
-import Tab from '@material-ui/core/Tab';
-import AppBar from '@material-ui/core/AppBar';
-// import Typography from '@material-ui/core/Typography';
-import Box from '@material-ui/core/Box';
-import { makeStyles } from '@material-ui/core/styles';
-
+import Highlighter from 'react-highlight-words';
 import {
-  DataGrid,
-  GridColumnsToolbarButton,
-  GridToolbarExport,
-} from '@material-ui/data-grid';
```

Comment:
Glad to see all these MUI artifacts being dropped - cleans up our code significantly!